src/UploadHandler.cpp

```diff
@@ -111,11 +109,11 @@ bool UploadHandler::createFile(HTTPRequest &request)
 		{
 			outfile << it->fileContent;
 			outfile.close();
-			std::cout << "File created successfully at " << filePath << std::endl;
+			Debug::log("File created successfully at " + filePath, Debug::NORMAL);
```

Comment:
Probably all these Debug::NORMAL should be Debug::ERROR and we will always print ERROR (proabably in red). See my http branch for this enhancement. Debug::ERROR could add always RED.

Reply:
Then gonna do it in your PR